lib/xcodeproj/workspace.rb

```diff
@@ -67,9 +89,36 @@ def self.from_s(xml, workspace_path = '')
     # @return [void]
     #
     def <<(projpath)
```

Comment:
Apologies for my ignorance -- is this proper syntax for documenting a parameter with two allowed types? The yard @overload documentation seems cumbersome for something like this...

    # Adds a new path to the list of the of projects contained in the
    # workspace.
    # @param [String, Xcodeproj::Workspace::FileReference] path_or_reference
    #        A string or Xcode::Workspace::FileReference containing a path to an Xcode project
    #
    # @raise [ArgumentError] Raised if the input is neither a String nor a FileReference
    #
    # @return [void]
    #
    def <<(path_or_reference)